src/Components/Background/index.ts

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+interface BackgroundProps {
+    imageUrl: string
```

Comment:
I get the following warning in the console:
```
Warning: React does not recognize the `imageUrl` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `imageurl` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```
You have to change prop name to be lowercase